src/applications/disability-benefits/all-claims/pages/medicalCareHistory.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+const medicalCareDescription = (
+  <div>
+    <h5>Medical care</h5>
+    <p>
+      During the last 12 months, have you been under a doctor's care or
+      hospitalized for these disabilities?
+    </p>
+  </div>
+);
+
+export const uiSchema = {
+  'ui:title': ' ',
+  'ui:description': medicalCareDescription,
+  careQuestion: {
+    'ui:title': ' ',
+    'ui:widget': 'radio',
+    'ui:options': {
+      labels: {
+        yes: 'Yes',
+        no: 'No',
+      },
+    },
+  },
+  'view:careReceived': {
+    'ui:options': {
+      expandUnder: 'careQuestion',
+      expandUnderCondition: 'yes',
+    },
+    medicalTreatment: {
+      'view:doctorCare': {
+        'ui:title': 'Yes, I‘ve been under doctor‘s care',
+      },
+      'view:hospitalization': {
+        'ui:title': 'Yes, I‘ve been hospitalized',
+      },
+    },
+  },
+};
+
+export const schema = {
+  type: 'object',
+  properties: {
+    careQuestion: {
```

Comment:
Should `careQuestions` be required?

Reply:
@Jennywalter : Is this required?

Reply:
@imeecuison,  the Yes/No question needs to be required.  Then at least one of the 'Yes' sub-options should be required.

Reply:
I made these required. Thank you!